### ISSUE TITLE
can you guys support eureka server starter

### ISSUE THREAD

**Issue by RobinsChens** [fabric8io/spring-cloud-kubernetes#71]
can you guys support eureka server starter

**Comment by iocanel** [comment]
We haven't added anything eureka, as for kubernetes we provide an alternative discovery client based on Kubernetes services.

I wouldn't mind adding eureka into the mix, as long as there is a real need. So can you please provide some more details on what you need?

**Comment by jstrachan** [comment]
linkerd might be a nicer option for more flexible load balancing options: https://linkerd.io/

**Comment by RobinsChens** [comment]
@iocanel I greatly appreciate it for your reply,our teams want to use spring-cloud with eurake,config-server,stream components .e.g

**Comment by iocanel** [comment]
It may make sense to add manifests for eureka inside kubeflix and add here a custom module for using the kubernetes powered eureka server.

It needs some thought. Meanwhile any additional information on your use case (and maybe a few words, on why you can't just use the current discovery module) will be more than welcome.

**Comment by RobinsChens** [comment]
@jstrachan but I need manage configuration unified

**Comment by RobinsChens** [comment]
@iocanel  yeah,that's a nice idea to add manifests for eureka inside kubeflix,I will have a try  ,thanks for your reply

**Comment by RobinsChens** [comment]
@iocanel  DId you manange configuration with   "Archaius Bridge",I want to have a demo to integrate to my project,I will appreciate it if you can show me somet demo or test

**Comment by RobinsChens** [comment]
@iocanel  DId you manange configuration with   "Archaius Bridge",I want to have a demo to integrate to my project,I will appreciate it if you can show me somet demo or test
